deploy/README.md

```diff
@@ -104,8 +81,8 @@ Here's an example with values filled in:
 
 ```yaml
 git:
-  URL: git@github.com:squaremo/flux-testdata
-  path: testdata
+  URL: git@github.com:squaremo/flux-example
```

Comment:
I do not; I'm giving an example of a fork which I've made for the purpose of trying it out, as suggested above.